projects/js-packages/publicize-components/src/components/block-editor/shared-utils.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file contains the utility functions that are shared between Jetpack and Social sidebar entrypoints
+ */
+import { dispatch } from '@wordpress/data';
+import domReady from '@wordpress/dom-ready';
+import { store as interfaceStore } from '@wordpress/interface';
+import { store as socialStore } from '../../social-store';
+
+export class JetpackSidebarManager {
+	url: URL;
+
+	queryArg = 'jetpack-sidebar';
```

Comment:
I'm not sure we need all of this now, and it's not really about the sidebar. What about adding a function to the shared utils package, which gets the value a more generic parameter like `jetpack-editor-action`, and then if that equals `reshare_post` it opens the modal?

Reply:
Done.